### DISCUSSION TITLE
Audio/Video (A/V) with captions and/or transcripts

### DISCUSSION THREAD

**Discussion by lhenze** [ProjectMirador/mirador#3537] [Looking for examples]
Hello! Wondering if anyone has examples of Mirador being used as an A/V player, with the feature of showing captions and viewing a transcript. Many thanks.

**Comment by lutzhelm** [comment]
I'm afraid the [demo](https://mirador-dev.netlify.app/__tests__/integration/mirador/video.html) for videos with captions isn't currently working due to changes in the cookbook recipe manifest. I summarized the problem in #3538. You could get captions to work in Mirador if you provided them in the same way the original cookbook recipe preview did, but this would go against the pattern suggested in the cookbook recipe.
